packages/neuron-ui/src/widgets/Dot/dot.module.scss

```diff
@@ -0,0 +1,16 @@
+@import '../../styles/theme.scss';
+
+.dot {
```

Comment:
Does `dot` mean a simple badge as https://m3.material.io/components/badges/overview
![image](https://github.com/nervosnetwork/neuron/assets/7271329/f42627a8-1872-467d-8692-7e910fec7cbc)

Reply:
> Does `dot` mean a simple badge as https://m3.material.io/components/badges/overview

I think it's the same as a badge. And I try to avoid using external UI component libraries.

Reply:
Ah I'm not saying the standalone implementation is not good, the point is that the name is not straight, `badge` or something is more clear than `dot`

Reply:
> Ah I'm not saying the standalone implementation is not good, the point is that the name is not straight, `badge` or something is more clear than `dot`

Oh, got it.